pyproject.toml

```diff
@@ -117,6 +119,7 @@ pytest-xdist = {extras = ["psutil"], version = "^3.6.1"}
 [tool.ruff]
 target-version = "py310"
 line-length = 88
+extend-exclude = [ "b2b/keycloak_admin_dataclasses.py" ]
```

Comment:
Was this temporary or is there a reason to exclude the entire file from formatting/linting?